server/models/Request.js

```diff
@@ -0,0 +1,24 @@
+import mongoose from "mongoose";
+
+const { Schema, model } = mongoose;
+
+const requestSchema = new mongoose.Schema(
+  {
+    customer: { type: Schema.Types.ObjectId, ref: "User", required: true },
+    provider: { type: Schema.Types.ObjectId, ref: "User", required: true },
+    start: { type: Date, required: true },
+    end: { type: Date, required: true },
+    accepted: { type: Boolean, default: false },
+    declined: { type: Boolean, default: false },
+    paid: { type: Boolean, default: false },
+    serviceType: {
+      type: String,
+      enum: ["Boarding", "House Sitting", "Day Care", "Walking"],
+    },
+    totalPrice: { type: Number, required: true },
```

Comment:
On one hand I think this is a nice addition. On the other hand, the most common practice for this would be, say, having an `Invoice` model and provide a `ref` here... But considering the scope of the project, it might be better to leave this in and not complicate things in favor of time/completion. Just wanted to share thoughts 👀 👍🏼 

Edit: I remembered there is a `paid` field here too so maybe it's best to do it this way for now 👍🏼

Reply:
Thanks, the reason to include `totalPrice` arose from the payment feature where I think we'll need this amount. Another model for Invoice sounds better, but following your advice, I'm gonna leave it like that.